### PR TITLE
Add  to aws-china.md to prevent Etag not match the HASH of file.

### PR DESCRIPTION
**Pull request by leeeboo**
I follow the docs [https://github.com/leeeboo/kops/blob/master/docs/aws-china.md#offline-mode](url) to build cluster offline in cn-north-1.
I find a bug in the docs. When sync files to s3, aws cli will upload multipart (8MB per one part). It will let the Etag which s3 returns not match the md5 of the file. So the cluster can not startup.

### CODE REVIEW

**Comment by chrislovecnm** [comment]
@leeeboo let @rdrgmnzs take a look first and then I will approve it

File: docs/aws-china.md [hunk 1]
```diff
@@ -254,10 +254,25 @@ done
...
+
+AWS_S3_DEFAULT_MULTIPART_THRESHOLD=$(aws configure get default.s3.multipart_threshold)
+
+if [ ! -n "$AWS_S3_DEFAULT_MULTIPART_THRESHOLD" ]; then
+  AWS_S3_DEFAULT_MULTIPART_THRESHOLD=8MB
+fi
+
+## Set multipart_threshold to 1024MB to prevent Etag not returns MD5 when upload multipart
+
+aws configure set default.s3.multipart_threshold 1024MB
```
Comment (justinsb) [inline docs/aws-china.md]: I feel there must be a way to specify this only for this instance, but I looked and I couldn't find it either!

**Comment by justinsb** [comment]
/ok-to-test

/lgtm

Thanks @leeeboo :-)

**Pull request merged**